src/test/java/org/folio/circulation/support/http/client/OffsetTest.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.circulation.support.http.client;
+
+import static org.folio.circulation.support.http.client.Offset.noOffset;
+import static org.folio.circulation.support.http.client.Offset.offset;
+import static org.folio.circulation.support.http.client.PageLimit.limit;
+import static org.folio.circulation.support.http.client.PageLimit.noLimit;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class OffsetTest {
+  @Test
+  public void shouldCalculateOffsetWhenPageLimitAndOffsetValuesNonNull() {
+    final Offset offset = offset(14).nextPage(limit(7));
+
+    assertEquals(offset.getOffset(), 21);
```

Comment:
Unlike Hamcrest, JUnit `assertEquals` seems to want the expected value first. These assertions are reported as code smells by Sonar.